mysql-test/suite/innodb/r/alter_copy_bulk,OFF.rdiff

```diff
@@ -1,5 +1,5 @@
---- bulk_copy_alter.result
-+++ bulk_copy_alter,non_bulk_alter_copy.result
+--- alter_copy_bulk.result
++++ alter_copy_bulk,OFF.result
```

Comment:
This change seems to be unnecessary. The `---` and `+++` lines only need to exist at the start of the file; the exact contents should not matter.